packages/remix-react/errorBoundaries.tsx

```diff
@@ -69,11 +69,9 @@ export function RemixRootDefaultErrorBoundary({ error }: { error: unknown }) {
   if (isRouteErrorResponse(error)) {
     return (
       <BoundaryShell title="Unhandled Thrown Response!">
-        <main style={{ fontFamily: "system-ui, sans-serif", padding: "2rem" }}>
-          <h1 style={{ fontSize: "24px" }}>
-            {error.status} {error.statusText}
-          </h1>
-        </main>
```

Comment:
Moved the `main` element into `BoundaryShell` so it's consistent across all default root component implementations